crates/env/src/engine/on_chain/impls.rs

```diff
@@ -357,7 +357,7 @@ impl TypedEnvBackend for EnvInstance {
         self.get_property_inplace::<T::AccountId>(ext::caller)
     }
 
-    fn transferred_balance<T: Environment>(&mut self) -> T::Balance {
+    fn transferred_value<T: Environment>(&mut self) -> T::Balance {
```

Comment:
If we really wanted to nitpick on the consistency then the `seal` fn `value_transferred` should also be the same :grinning: